rest-api/add-gif-picker/server/package.json

```
@@ -0,0 +1,16 @@
+{
+    "scripts": {
+        "start": "node server.js"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "type": "module",
+    "dependencies": {
+        "axios": "^1.5.1",
+        "body-parser": "^1.20.2",
+        "cors": "^2.8.5",
+        "dotenv": "^16.3.1",
+        "express": "^4.18.2",
```

```suggestion
        "express": "^4.18.2"
```

I got an error and had to remove this